benchmarks/apps/sombrero/sombrero.py

```diff
@@ -5,12 +5,24 @@
 
 import reframe as rfm
 import reframe.utility.sanity as sn
-import reframe.utility.udeps as udeps
 
 from benchmarks.apps.sombrero import case_filter
 from benchmarks.modules.reframe_extras import scaling_config
 from benchmarks.modules.utils import SpackTest
 
+# Fixed lattice volume in ITT benchmarks
+LATTICE_VOLUME = 32 * 24 * 24 * 24
```

Comment:
Looking at [the Sombrero README](https://github.com/sa2c/sombrero#problem-sizes), I think this is only the lattice volume in the case `-s small` is passed to the executable. For the 64 node test, `-s medium` is currently passed so the lattice volume is $48^3 \cdot 64$. It's possible these might be changed as well in order to resolve #246.

Having said that, I think this will probably still work fine since the prime factor decomposition of these numbers are both of the form $2^p3^q$ and they're much bigger than the number of cores on even the largest nodes.

Reply:
Thanks, I defined `LATTICE_VOLUME_SMALL` and `LATTICE_VOLUME_MEDIUM` to distinguish the two cases.